src/pipx/commands/install.py

```diff
@@ -40,44 +42,55 @@ def install(
         exists = False
 
     venv = Venv(venv_dir, python=python, verbose=verbose)
-    if exists:
-        if force:
-            print(f"Installing to existing venv {venv.name!r}")
-        else:
-            print(
-                pipx_wrap(
-                    f"""
-                    {venv.name!r} already seems to be installed. Not modifying
-                    existing installation in {str(venv_dir)!r}. Pass '--force'
-                    to force installation.
-                    """
-                )
-            )
-            return EXIT_CODE_INSTALL_VENV_EXISTS
 
-    try:
-        venv.create_venv(venv_args, pip_args)
-        venv.install_package(
-            package_name=package_name,
-            package_or_url=package_spec,
+    if upgrade:
+        upgrade_package(
+            venv_dir=venv_dir,
             pip_args=pip_args,
-            include_dependencies=include_dependencies,
-            include_apps=True,
-            is_main_package=True,
-            suffix=suffix,
-        )
-        run_post_install_actions(
-            venv,
-            package_name,
-            local_bin_dir,
-            venv_dir,
-            include_dependencies,
             force=force,
+            include_injected=True,
+            verbose=verbose,
         )
```

Comment:
We should probably keep installing the package even if it is not installed. (Currently it will exit with an error when the package is not installed.)

Reply:
The change is placed. I had to add `-f` to the test however because the assert won't work with existing installations.

Reply:
I think `--force` should automatically be passed when upgrading?

Reply:
Yeah, I agree with this.

Reply:
Done.